_posts/patterns/2015-04-15-icons.md

```diff
@@ -372,35 +372,35 @@ info:               Icons are used to clearly and visually represent some inform
             </span>
         </div>
         <div class="icon-classname">
-            Close, Remove
+            close, remove
         </div>
     </div>
     <div class="example example-icon">
         <div class="icon-display">
             <span class="icon-fallback icon-fallback-text">
-                <span class="icon icon-times-circle-o" aria-hidden="true"></span>
+                <span class="icon icon-close-circle" aria-hidden="true"></span>
                 <span class="text">Remove</span>
             </span>
         </div>
         <div class="icon-classname">
-            Close, Remove (light)
+            close-circle
         </div>
     </div>
     <div class="example example-icon">
         <div class="icon-display">
             <span class="icon-fallback icon-fallback-text">
-                <span class="icon icon-times-circle" aria-hidden="true"></span>
+                <span class="icon icon-close-circle-light" aria-hidden="true"></span>
                 <span class="text">Remove</span>
             </span>
         </div>
         <div class="icon-classname">
-            Close, Remove (dark)
+            close-circle-light
         </div>
     </div>
     <div class="example example-icon">
         <div class="icon-display">
             <span class="icon-fallback icon-fallback-text">
-                <span class="icon icon-exclamation-circle" aria-hidden="true"></span>
+                <span class="icon icon-error" aria-hidden="true"></span>
                 <span class="text">Error</span>
             </span>
         </div>
```

Comment:
I can't comment on [408](https://github.com/edx/ux-pattern-library/pull/284/files#diff-8e6a0a47e453c2074d1a3b7fbd3c0280R408), but it looks like that should be updated to the lowercase name now.